kolibri/plugins/coach/assets/src/views/new/shared/status/InProgress.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+
+  <LabeledIcon :label="text">
+    <mat-svg category="device" name="access_time" />
+  </LabeledIcon>
+
+</template>
+
+
+<script>
+
+  import statusMixin from './status';
+
+  export default {
+    name: 'InProgress',
+    mixins: [statusMixin],
+    $trs: {
+      basicLabelInProgress: '{count, plural, other {In-progress}}',
```

Comment:
Is the hyphen here really necessary?

Reply:
addressed - 558dbbc0b59e95340fc7ae427ee3b79e7580f0b8